README.md

```diff
@@ -2,41 +2,39 @@
 
 [![Build](https://travis-ci.org/auth0/express-jwt.png)](http://travis-ci.org/auth0/express-jwt)
 
-Middleware that validates JsonWebTokens and sets `req.user`.
-
-This module lets you authenticate HTTP requests using JWT tokens in your Node.js
-applications.  JWTs are typically used to protect API endpoints, and are
-often issued using OpenID Connect.
+This module provides Express middleware for validating and decoding JWTs ([JSON Web Tokens](https://jwt.io)) through the [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken/) module. The decoded information is provided on the Express request object.
 
 ## Install
 
-    $ npm install express-jwt
+```
+$ npm install express-jwt
+```
 
 ## Usage
 
-The JWT authentication middleware authenticates callers using a JWT.
-If the token is valid, `req.user` will be set with the JSON object decoded
-to be used by later middleware for authorization and access control.
-
-For example,
+Basic usage using an HS256 secret:
```

Comment:
@aarongodin is the default behavior to always inject it as a `user` property in the `request` object or is that something the user can change? If it's cannot be changed, I think it's important to mention that the info is available on `req.user` 🤔

Reply:
Good point. That is the default behavior, but it is customizable. I made a note about the default behavior below the first example, and I linked to a section heading I created.